### PR TITLE
Fix layout to follow bc CSS

### PR DESCRIPTION
**Pull request by rgaiacs**
This is related with #12 and try to address [my comment](https://github.com/swcarpentry/lesson-template/pull/12#issuecomment-62623131).

swc

### CODE REVIEW

File: _includes/banner.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<span id="github-ribbon"><a href="https://github.com/swcarpentry/bc">Fork me on GitHub</a></span>
```
Comment (gvwilson) [inline _includes/banner.html]: Nope - we definitely don't want to point people at the old 'bc' repo any longer.

File: _layouts/page.html [hunk 2]
```diff
@@ -2,14 +2,21 @@
...
+    <div class="container">
+      $banner$
+      <div class="row-fluid">
+        <div class="span10 offset1">
+          $if(title)$<h1>$title$</h1>$endif$
+          $body$
+        </div>
+      </div>
+      $footer$
+    </div>
```
Comment (gvwilson) [inline _layouts/page.html:19]: +1 to this change.

**Comment by gvwilson** [comment]
@r-gaia-cs does this depend on #12, or can they be merged separately?

File: _includes/banner.html (hunk 1 above)
Comment (rgaiacs) [inline _includes/banner.html]: I didn't have much time to think how set the properly address. Do you have any suggestion?

The quickly solution was add another variable to pass for Pandoc.

**Comment by rgaiacs** [comment]
@gvwilson It can be merged separately. After you merge this and #12 lessons will have the "old" look. To avoid problems you can merge #12 and later I will send another PR cleaning it.

**Comment by gvwilson** [comment]
OK: please go ahead and do the merges without the "fork me on GitHub" 
banner bit, file an issue for us to figure that out, and we'll re-add it 
later.  (I'm thinking about ways to use Git commands to automatically 
get the right URL for the master copy of the lesson, but I think it's 
trickier than you might think, because the principal author's remote is 
probably 'origin', not 'upstream'.)

**Pull request merged**